releasenotes/config.yaml

```diff
@@ -1,6 +1,16 @@
 ---
 default_branch: origin/1.x
 unreleased_version_title: Unreleased
+sections:
+  # The prelude section is implicitly included.
+  - [upgrade, Upgrade Notes]
+  - [features, New Features]
+  - [issues, Known Issues]
+  - [api, API Changes]
+  - [security, Security Issues]
+  - [fixes, Bug Fixes]
+  - [other, Other Changes]
+  - [deprecations, Deprecation Notes]
```

Comment:
let's move deprecations up to under upgrade -- we want people to stop using deprecated features